src/main/java/com/nirima/jenkins/plugins/docker/listener/DockerQueueListener.java

```diff
@@ -28,24 +28,37 @@
 public class DockerQueueListener extends QueueListener {
 
     @Override
-    public void onEnterWaiting(WaitingItem wi) {
+    public void onEnterWaiting(final WaitingItem wi) {
         final DockerJobTemplateProperty jobTemplate = getJobTemplate(wi);
         if (jobTemplate != null) {
-            final DockerCloud cloud = DockerCloud.getCloudByName(jobTemplate.getCloudname());
+            final DockerCloud cloud = 
+                DockerCloud.getCloudByName(jobTemplate.getCloudname());
             if (cloud != null) {
                 final String uuid = UUID.randomUUID().toString();
-                final DockerTemplate template = jobTemplate.getTemplate().cloneWithLabel(uuid);
+                final DockerTemplate template = 
```

Comment:
Please use `mvn spotless:apply` rather than performing manual formatting of the files.  The Jenkins plugin parent pom provides spotless with a configuration that works well for Jenkins plugins.  It is enabled in this plugin.